server.js

```diff
@@ -1,5 +1,51 @@
-const app = require("./app");
+const express = require("express");
+const logger = require("morgan");
+const cors = require("cors");
+const mongoose = require("mongoose");
+require("dotenv").config();
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000");
-});
\ No newline at end of file
+const app = express();
+
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
+
+app.use(logger(formatsLogger));
+app.use(cors());
+app.use(express.json());
+
+const contactsRouter = require("./routes/api/contacts");
+app.use("/api/contacts", contactsRouter);
+
+app.use((req, res, next) => {
+  res.status(404).json({
+    message: "Use api on routes: /api/contacts",
+    data: "Not found",
+  });
+});
+
+app.use((err, req, res, next) => {
+  res.status(500).json({
+    message: err.message,
+    data: "Internal Server Error",
+  });
+});
+
+const uriDb = process.env.DB_HOST;
```

Comment:
All capital with underscores in CONSTANTS!

Reply:
In the latest commit this was fixed by doing a refactor and as a result instead of the following 2 lines of code:
```
const uriDb = process.env.DB_HOST;
const PORT = process.env.PORT || 3000;
```

now there is one:
`const { PORT = 3000, DB_HOST: DB_URI } = process.env;`

Reply:
hej, slodziutko! :) well done